src/webextension/widgets/password-input.js

```diff
@@ -59,12 +59,12 @@ export default class PasswordInput extends React.Component {
                ref={(element) => this.inputElement = element} {...props}/>
         <ButtonStack selectedIndex={selectedIndex}>
           <Localized id="password-input-show">
-            <button type="button" disabled={disabled}
-                    onClick={() => this.showPassword(true)}>sHOw</button>
+            <button className={styles.showBtn} type="button" title="sHOw" disabled={disabled}
+              onClick={() => this.showPassword(true)}></button>
```

Comment:
Nit: could you restore the indentation here? `onClick` should be directly below `className`.